Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -987,6 +987,21 @@ a fixed cookie value as a trade-off. This makes all applications on the host to
 select the same service endpoint for a given service with session affinity configured.
 To disable the feature, set ``config.sessionAffinity=false``.
 
+LoadBalancer Source Ranges Checks
+*********************************
+
+When a ``LoadBalancer`` service is configured with ``spec.loadBalancerSourceRanges``,
+Cilium's eBPF kube-proxy replacement restricts access from outside to the service
+to the IPs specified in the field. If the field is empty, no restrictions for
+the access will be applied.
+
+When accessing the service from inside a cluster, the kube-proxy replacement will
+ignore the field regardless whether it is set. This means that any pod or any host
+process in the cluster will be able to access the ``LoadBalancer`` service.
+
+The loadbalancer source range check is enabled by default, and it can be disabled
+by setting ``config.svcSourceRangeCheck=false``.
+
```

Comment:
Maybe this should also add a paragraph that some public cloud providers who offer LB service type already implement this for external traffic.